docs/feature-guides/clarinet-deploy.md

```diff
@@ -4,49 +4,36 @@ title: Clarinet Deployment Plans
+Deployment Plans are reproducible deployment steps that publish a collection of on-chain transactions and one or more contracts to a network, whether a local developer network, the public testnet, or into production on mainnet. Deployment plans minimize the inherent complexity of deployments, such as smart contract dependencies and interactions, transaction chaining limits, deployment costs, and more, while ensuring reproducible deployments critical for testing purposes.
 
 ## Design
 
-Deployment Plans are reproducible deployment steps that publish a “protocol”; a collection of on-chain transactions and contracts—to a network, whether it is on a local developer network, the public testnet, or into production on mainnet.
-
-Using a Deployment Plan can minimize the inherent complexity with deployments, thereby making it much easier to deploy a contract without errors. Some of these complexities can include dependencies, chaining limits, the process to initialize a smart contract, and underlying deployment costs.
-
-A deployment plan is made up of:
-
-- smart contracts
-- accounts with their token balances, and
-- content and sequence of transactions (across a single or multiple Stacks or Bitcoin blocks)
-
-A Deployment Plan’s specifications exist on two files within a Clarinet project: the `Clarinet.toml` and network’s `.toml` file (for example, `devnet.toml`) under the “deployments” folder.
+The default deployment plan of every Clarinet project is contained within specifications set inside certain files. In addition to this default deployment plan, the user can manually configure each plan, adding additional transactions or contract calls, across multiple Stacks or Bitcoin blocks.
 
 You can commit, audit, and test contracts without including any secrets in the Deployment Plan, and share these contracts without exposing any sensitive information.
 
 ## Deployment plan primitives
 
-Deployment plans consist of the following primitives:
+Deployment plans consist of the following transaction primitives:
 
-- deploy contracts
+- publish contracts
 - call contracts
 - send bitcoin transactions
 - wait for block
 - send stacks to an address or contract
 
-With these four individual primitives, you can then:
+With these transaction primitives, you can then:
 
-- Deploy a contract in an in-memory simulated chain (simnet only). 
-- Call a contract that has been deployed in an in-memory simulated chain (simnet only).
-- Deploy an external contract on another testnet / devnet network using another wallet + search, and replace all references to this contract in the local contracts to deploy (devnet / testnet only).
-- Deploy a contract (devnet / testnet / mainnet).
-- Call a contract (devnet / testnet / mainnet).
-- Perform a simple bitcoin transfer from a p2pkh address to a p2pkh address (experimental, regtest / testnet / mainnet).
+- Deploy a contract in an in-memory simulated chain (simnet only)
```

Comment:
@mefrem: Would it be helpful to convert the primitives and the scenarios into a 2-column table, so it would be easy to compare the primitive and the usage of it?

Reply:
I don't disagree! Just made such a table